app/controllers/publishers_controller.rb

```diff
@@ -35,22 +37,18 @@ class PublishersController < ApplicationController
     only: %i(verification_dns_record
              verification_public_file)
 
-  def new
-    @publisher = Publisher.new
-    @should_throttle = should_throttle_create?
-  end
-
   def create
-    @publisher = Publisher.new(publisher_create_params)
+    @publisher = Publisher.new(pending_email: params[:email])
+
     @should_throttle = should_throttle_create?
     throttle_legit =
       @should_throttle ?
         verify_recaptcha(model: @publisher)
         : true
     if throttle_legit && @publisher.save
-      # TODO: Change to #deliver_later ?
-      PublisherMailer.welcome(@publisher).deliver_later!
-      PublisherMailer.welcome_internal(@publisher).deliver_later if PublisherMailer.should_send_internal_emails?
+      PublisherMailer.verify_email(@publisher).deliver_later!
+      PublisherMailer.verify_email_internal(@publisher).deliver_later if PublisherMailer.should_send_internal_emails?
+      session[:created_publisher_id] = @publisher.id
       session[:created_publisher_email] = @publisher.email
```

Comment:
sorry I didn't catch this – this should probably be `@publisher.pending_email`

feel free to update this in a small patch after